src/lib/worker.js

```diff
@@ -1,42 +1,83 @@
 import path from 'path';
+import {spawn} from 'child_process';
 
-const testOptions = JSON.parse(process.env.testOptions);
-const cucumber = require(testOptions.cucumberPath).Cli;
-
-const featureFile = process.env.featureFile;
-const featureFileData = path.parse(process.env.featureFile);
-const featureName = featureFileData.name;
-const scenarioLine = process.env.scenarioLine;
-const logFileName = featureName + '-line-' + scenarioLine + '.json';
-const logFile = path.join(testOptions.logDir, logFileName);
-const relativeLogFile = path.relative(process.cwd(), logFile);
-
-let args = ['', '', featureFile + ':' + scenarioLine, '-f', 'json:' + relativeLogFile];
-
-testOptions.requires.forEach(function(arg) {
-  args.push('-r');
-  args.push(arg);
-});
-
-let startTime = new Date();
-
-try {
-  cucumber(args).run(function(isSuccessful) {
-    let exitCode = (isSuccessful) ? 0 : 1;
-    process.send({
-      type: 'result',
-      exitCode: exitCode,
-      resultFile: logFile,
-      duration: new Date() - startTime
+export default class Worker {
```

Comment:
Having a constructor actually do work instead of just initializing variables doesn't seem like the best thing to me.